assets/_sass/components/_social-share-icons.scss

```diff
@@ -0,0 +1,69 @@
+/*==================================
+=       Social Share Icons         =
+==================================*/
+
+.share {
+    $s: &;
+    $i: icon-facebook, icon-twitter, icon-email, icon-print;
+    display: flex;
+    padding: .5rem;
+
+    &__text {
+        @include btn;
+        text-transform: uppercase;
+        padding-bottom:.2rem;
+    }
+
+    ul {
+        list-style-type: none;
+        display: flex;
+        justify-content: space-between;
+        margin: 0;
+        padding: 0;
+
+        li {
+            display: inherit;
+            margin: 0;
+            padding-right: 0.1rem;
+            text-align: center;
+            line-height: 1;
+
+            i {
+                font-size: 1.3rem;
+            }
+        }
+    }
+
+    .page-header & {
+        background-color: $color-dust;
+
+        #{$s}__text {
```

Comment:
This is the right idea, but this selector isn't valid because `share__text` is a class, not an ID.

Reply:
For some reason - this works. I was selecting a 'grandparent' selector, and this seems to be the syntax? https://codeburst.io/how-to-do-sass-grandparent-selectors-b8666dcaf961

Reply:
Got it, I see now. The `#` in this case isn't an ID, it's part of the SCSS interpolation of the variable `#{$VARIABLE}`. Great, you can disregard these two notes then.